userdocs/docs/grpc/services/microvm/v1alpha1/proto.md

```diff
@@ -0,0 +1,210 @@
+# Protocol Documentation
```

Comment:
can you put somewhere how these docs files are generated and also how can i see them in the site? i cannot see a link to take me there

Reply:
ah actually i think this was something i forgot to remove when i was overhauling the docs